includes/class-wc-google-gtag-js.php

```diff
@@ -53,15 +55,28 @@ public function setup_frontend_scripts() {
 			// Filter variation data to include formatted strings required for add_to_cart event
 			add_filter( 'woocommerce_available_variation', array( $this, 'variant_data' ), 10, 3 );
 			// Add default inline product data for add to cart tracking
-			wp_enqueue_script( $this->script_handle );
+			wp_enqueue_script( $this->script_handle . '-ga-integration' );
 		}
 	}
 
 	/**
 	 * Register front end JavaScript
 	 */
 	public function register_scripts() {
-		wp_register_script( $this->script_handle, plugins_url( 'assets/js/ga-integration.js', dirname( __FILE__ ) ), array( 'jquery' ), WC_GOOGLE_ANALYTICS_INTEGRATION_VERSION, true );
+		wp_register_script(
+			$this->script_handle . '-ga-integration',
+			Plugin::get_instance()->get_js_asset_url( 'ga-integration.js' ),
+			Plugin::get_instance()->get_js_asset_dependencies( 'ga-integration' ),
```

Comment:
The 'jquery' handle is missing after changing to use `get_js_asset_dependencies`.

Reply:
I added a workaround here https://github.com/woocommerce/woocommerce-google-analytics-integration/commit/450963ebb6e324df566f5c56c7bf8eab298ef00d